graphql-dgs-spring-boot-oss-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/autoconfig/DgsAutoConfiguration.kt

```diff
@@ -236,4 +237,25 @@ open class DgsAutoConfiguration(
     open fun fluxReactiveDataFetcherResultProcessor(): FluxDataFetcherResultProcessor {
         return FluxDataFetcherResultProcessor()
     }
+
+    @Bean
+    open fun inputArgumentResolver(inputObjectMapper: ObjectProvider<InputObjectMapper>): ArgumentResolver {
+        val mapper = inputObjectMapper.ifAvailable ?: DefaultInputObjectMapper()
+        return InputArgumentResolver(mapper)
+    }
+
+    @Bean
+    open fun dataFetchingEnvironmentArgumentResolver(): ArgumentResolver {
+        return DataFetchingEnvironmentArgumentResolver()
+    }
+
+    @Bean
+    open fun coroutineArgumentResolver(): ArgumentResolver {
+        return ContinuationArgumentResolver()
+    }
+
+    @Bean
+    open fun fallbackDfeArgumentResolver(): ArgumentResolver {
```

Comment:
nit, why not name it `fallbackEnvironmentArgumentResolver` to be consistent with the above.